testsuite/dynaconf_loader.py

```diff
@@ -53,17 +52,13 @@ def _guess_version(ocp):
     """Attempt to determine version from amp-system imagestream"""
 
     version = None
-    lookup = _docker_image(ocp, "amp-system")
-
     try:
-        version = lookup["name"]
+        config = ocp.do_action("get", ["dc/apicast-staging", "-o", "yaml"], parse_output=True)
```

Comment:
original code did derivate version from `amp-system`... ???

Reply:
amp-system can have a different image stream tag if used with Oracle so I changed it to apicast instead.

Reply:
apicast-staging can have different image stream tag if custom-policy is injected (yeah, less probable). apicast-production?

Reply:
I don't think anyone is (and should be) manipulating apicast-staging deployment, however I will change it to apicast-production just to be safe

Reply:
Changed